calin/nz/_posts/2022-06-26-overview.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: The Start of a Journey
+author: Călin Cașcaval
+---
+
+
+While saying goodbye to a number of friends on the latest hike I organized for our
+BARD hiking group, someone suggested that I post stories and photos of our new
+planned adventure in New Zealand. So I decided to start journaling -- something that
+I have been considering for a while, but I did not have a good incentive. While we
```

Comment:
You could potentially trim this (and the next few sentences) just to call out the pictures, and get to the writing faster :)

Reply:
hmm, well, it is also part of the journey! I managed to avoid writing until now, and although I published pictures before, even captions and titles have been in short supply :)